source/blog/2017-11-07-fedora-atomic-27-on-multiarch.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Fedora 27 Atomic Host Availability on Multiple Architectures
+author: sinnykumari
+date: 2017-11-07
+published: true
+comments: false
+tags: atomic, fedora, multi-arch, ppc64le, aarch64
+published: true
+---
+
+We are proud to announce that multiple architectures are now supported with Fedora 27 Atomic Host release! Now, along with x86_64 architecture Atomic Host is also available on 64 bit ARM (AArch64) and PowerPC Little Endian (ppc64le). Both aarch64 and ppc64le architectures will receive Atomic OSTree updates in the same way x86_64 does.
+Atomic Host provides immutable operating system tree (OStree) with rollback facility. To learn more, see [Project Atomic website](http://www.projectatomic.io/).
+
+# Download
+Fedora Atomic Host for aarch64 and ppc64le are available in both ISOs and cloudImages-
+- aarch64
+    - [ISO](link)
+    - [QCOW](link)
+    - [Raw image](link)
+
+- ppc64le
+    - [ISO](link)
+    - [QCOW](link)
+    - [Raw image](link)
+
+# Installation
+Atomic Host can be installed as per user preference where they want to install it - on bare metal, virtual machine, cloud setup and so on.
+If you have some aarch64 or ppc64le hardware, we'd love for you to take Atomic Host for a spin. One quick way is to run `virt-install` on one of theses platforms:
+
+```
+$ sudo virt-install --name fedora-27-atomic --ram 2048 --vcpus 2 --disk path=/var/lib/libvirt/images/Fedora-Atomic-27.<arch>.qcow2 --os-type linux --os-variant fedora26 --network bridge=virbr0 --graphics vnc,listen=127.0.0.1,port=5901 --cdrom /var/lib/libvirt/images/init.iso --noautoconsole
+
+```
+Here, we have assumed that Fedora Atomic qcow2 image and init.iso file is available in /var/lib/libvirt/images/ directory.
+The file init.iso is metadata ISO to provide critical data when Atomic Host boots. Check out [Prep the cloud-init source ISO](http://www.projectatomic.io/docs/quickstart/) to easily create it.
+You might need to make other adjustments to the `virt-install` command line above depending upon your requirement.
+
+# Features
+Atomic Host provides various features, some of them are:
+* `Immutable Host`
+* `Package Layering` - Additional package(s) can be installed on top of base Atomic Host using package layering.
+```
+$ sudo rpm-ostree install elfutils
+```
+* `Running containers` - One can easily run a container image available locally or from a registry. For example, running hello-world container image from docker.io registry on ppc64le machine:
```

Comment:
Do we have any information on when FLIBS will have alt-arch containers?

Reply:
Checked with Adam Miller, FLIBS for multi-arch container image build is currently on hold.